draft-ietf-bidding-and-auction-services.md

```diff
@@ -733,10 +733,22 @@ response = {
   ;   5.       add a tuple to the list in (1) of [interest group owner (2),
                interest group name (4)]
   ;   6. return the list in (1)
-   ? biddingGroups: {
+  ? biddingGroups: {
     * interestGroupOwner => [* int]
   },
 
+  ; Indices and update-if-older-than times of interest groups in the original
+  ; request for this owner for interest groups where an update-if-older-than
+  ; time was specified.
+  ; Maps to https://wicg.github.io/turtledove/#server-auction-response-update-groups.

```

Comment:
is this meant to be a working link or is this forward looking?

Reply:
It's forward looking. The current link is here: https://pr-preview.s3.amazonaws.com/brusshamilton/turtledove/pull/1294.html#server-auction-response-update-groups